### PR TITLE
Medibase: Drop littletable; instead store as list from ORM's `values_list`

### PR DESCRIPTION
**Pull request by rithviknishad**
## Proposed Changes

- MedibaseMedicineTable no longer uses `littletable` and is now simply a `list` fetched ORM's `values_list`

## Merge Checklist

- [ ] Tests added/fixed
- [ ] Update docs in `/docs`
- [ ] Linting Complete
- [ ] Any other necessary step

_*Only PR's with test cases included and passing lint and test pipelines will be reviewed*_

@coronasafe/care-backend-maintainers @coronasafe/care-backend-admins

### CODE REVIEW

**Comment by nihal467** [comment]
I tested in  tunnel provided by @rithviknishad's and added multiple medicines to a patients looks good to me in a general view

**Comment by rithviknishad** [comment]
Worker timeout occurs as loading to in-memory on wsgi.py takes too much time for (0.25cpu)
No memory issues though.

**Comment by rithviknishad** [comment]
But this happens for me even with lazy load (not on wsgi.py, load on first search request) on my CPU.
Probably because my `0.25` cpu is not same as staging instances `0.25` cpu

File: care/facility/api/viewsets/prescription.py [hunk 1]
```diff
@@ -180,8 +172,6 @@ def list(self, request):
 
         if request.GET.get("query", False):
             query = request.GET.get("query").strip().lower()
```
Comment (sainak) [inline care/facility/api/viewsets/prescription.py:174]:
```suggestion
        if query := request.query_params.get("query"):
            query = query.strip().lower()
```

**Review by sainak — APPROVED** [review]

**Review by vigneshhari — APPROVED** [review]

**Pull request merged**